navidrome/docker-compose.yml

```
@@ -0,0 +1,13 @@
+services:
+  navidrome:
+    image: deluan/navidrome:latest
```

We try to use not the lastest tag to make it more clear which image tag the app uses. The digest at the end is needed to ensure reproducibility in case a tag gets changes (happens for the latest tag all the time)

```suggestion
    image: deluan/navidrome:0.52.5@sha256:b154aebe8b33bae82c500ad0a3eb743e31da54c3bfb4e7cc3054b9a919b685c7
```